### PR TITLE
Release Deb Package

### PR DESCRIPTION
**Pull request by isantop**
This PR adds in Deb packaging, which will allow users to download and install the package via Apt. 

This GNOME Shell extensions uses Touchegg to enable 1:1 workspace gesture tracking between the animation and the gesture. It's required to get workspace animation tracking on X11 systems.

Prior to release, this package should be tested for functionality and bugs. Namely, the following testing needs to pass:

* Install deb package `gnome-shell-extension-popx11gestures`
* Enable `COSMIC X11 Gestures` extension
* Ensure that system stability remains unchanged
* Ensure that on systems with multi-touch trackpads supporting at least four fingers, a four-finger up/down swipe correctly changes workspaces, and that the switch animation tracks 1:1 with the gesture input.

### CODE REVIEW

**Comment by linuxgnuru** [comment]
After installing gnome-shell-extension-popx11gestures got the following error:

```
/usr/share/glib-2.0/schemas/org.gnome.shell.extensions.popx11gestures.gschema.xml:11:6  Error on line 11 char 6: “<” is not a valid character following the close element name “description”
; the allowed character is “>”.  This entire file has been ignored.
```

**Comment by linuxgnuru** [comment]
on adding the missing > at the end of the file, tried to enable this but got the following error:

Screenshot from 2022-05-16 15-59-37

**Comment by jacobgkau** [comment]
@linuxgnuru I'm seeing it install and enable successfully after the last change to close that XML tag (although it should probably get enabled by default, and it wasn't.)

**Comment by jacobgkau** [comment]
Actually, getting enabled by default would probably be done in pop-session: https://github.com/pop-os/session/blob/master_jammy/usr/share/gnome-shell/modes/pop.json

**Comment by jacobgkau** [comment]
This is what the new package name looks like next to existing extensions starting with `gnome-shell-extension-pop`:

Screenshot from 2022-05-16 16-43-53

Is there a reason it's not `gnome-shell-extension-pop-x11-gestures` or at least `gnome-shell-extension-pop-x11gestures`?

**Comment by isantop** [comment]
@linuxgnuru Adding the `>` itself doesn't fix the schema, you need to recompile the schemas (or update the debpackage, which does that for you).

@jacobgkau 

> Is there a reason it's not gnome-shell-extension-pop-x11-gestures 

The upstream extension is just `x11gestures`, so splitting that up doesn't seem appropriate to me.

> or at least gnome-shell-extension-pop-x11gestures

Originally it was to keep the naming style consistent with the upstream. It can be changed, but we don't currently have guidelines about package names that are expected to be adhered to, so I didn't think it was a huge issue. These generally aren't going to be interacted with by end-users anyway.

**Comment by isantop** [comment]
Enabling by default is something that should be updated in Pop Session, as those files are controlled by that package rather than the individual extension packages.

File: popx11gestures@system76.com/src/touchegg/ToucheggClient.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
+/*
```
Comment (jackpot51) [inline popx11gestures@system76.com/src/touchegg/ToucheggClient.js]: I am somewhat sure this directory needs to be in .gitignore

**Comment by jacobgkau** [comment]
> Originally it was to keep the naming style consistent with the upstream. It can be changed, but we don't currently have guidelines about package names that are expected to be adhered to, so I didn't think it was a huge issue. These generally aren't going to be interacted with by end-users anyway.

I see the upstream is called `gnome-shell-extension-x11gestures`, it would be nice if we could at least separate Pop out from the extension name so it's `gnome-shell-extension-pop-x11gestures`. I understand this obviously won't be seen by users very often, but it would be nice to be consistent. I assume it's easier/less risky to change now than after we start shipping it, that's the only reason I brought it up.

**Review by linuxgnuru — APPROVED** [review]
worked without issues. If touchpad is disabled gestures still work but issue with touchegg not this extension

**Review by leviport — CHANGES_REQUESTED** [review]
Let's get this renamed to `gnome-shell-extension-pop-x11gestures`, as discussed above. John showed me that it's working quite well, so I'd love to see it released soon.

File: popx11gestures@system76.com/COPYRIGHT [hunk 2]
```diff
@@ -0,0 +1 @@
+Copyright (C) 2021 - 2022 José Expósito <jose.exposito89@gmail.com>
```
Comment (jackpot51) [inline popx11gestures@system76.com/COPYRIGHT]: Just want to mention again, I think this folder popx11gestures@system76.com should not be checked in, and should be in the .gitignore

**Review by jackpot51 — CHANGES_REQUESTED** [review]

**Review by linuxgnuru — APPROVED** [review]
Package was renamed and still functions as it intended (and is buttery smooth)

**Review by leviport — APPROVED** [review]
Approved per John's testing

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**